### PR TITLE
Owners can no longer remove themselves

### LINKED ISSUE #78: `IdentityManager::removeOwner()` and `MetaIdentityManager::removeOwner()` may be called on oneself
https://github.com/uport-project/uport-identity/blob/880f8b81b9355edfe62c35817562b24e90513d2a/contracts/IdentityManager.sol#L128-L132

* [IdentityManager::removeOwner()](https://github.com/uport-project/uport-identity/blob/880f8b81b9355edfe62c35817562b24e90513d2a/contracts/IdentityManager.sol#L128-L129)

As either `removeOwner()` function is written, an older owner has the ability to remove oneself as an owner. This functionality increases the likelihood that an owner accidentally deletes itself.

**Recommendation**

Add `require()` statement in both functions that ensures that `removeOwner()` cannot be called on oneself.

### PR DESCRIPTION
**Pull request by oed**
This fixes #78

### CODE REVIEW

**Comment by naterush** [comment]
Is it possible there is some situation where an owner wants to remove themselves? Consider the following situation. 

There are two owner keys. The first one performed some command (say, removing a different owner key) and thus is currently cooling down. The second one is compromised - in the sense that someone else gets control of the key as well. If the user cannot use the owner key to remove itself, then the adversary has more time to delete the other currently-frozen owner key.

It seems to me this is something we want to enforce on a UX level, rather than in the contracts, based on situations like the above (though it is totally contrived).

**Comment by naterush** [comment]
Will add this to the issue.

File: contracts/MetaIdentityManager.sol [hunk 1]
```diff
@@ -150,6 +150,8 @@ contract MetaIdentityManager {
         onlyOlderOwner(identity, sender)
         rateLimited(identity, sender)
     {
+        // an owner should not be allowed to remove itself
+        require(msg.sender != owner);
```
Comment (maurelian) [inline contracts/MetaIdentityManager.sol]: Should this not be `require(sender != owner);` to allow meta tx?

Comment (oed) [reply contracts/MetaIdentityManager.sol]: yep, thanks!

File: contracts/MetaIdentityManager.sol [hunk 2]
```diff
@@ -151,7 +151,7 @@ contract MetaIdentityManager {
         rateLimited(identity, sender)
     {
         // an owner should not be allowed to remove itself
-        require(msg.sender != owner);
+        require(sender != owner);
```
Comment (maurelian) [inline contracts/MetaIdentityManager.sol]: LGTM

**Pull request merged**